### DISCUSSION TITLE
i get  -  read: connection reset by peer

### DISCUSSION THREAD

**Discussion by cskml** [jackc/pgx#1469] [Q&A]
go mod init hello 

go get github.com/jackc/pgx/v5

why i get this?

go: module github.com/jackc/pgx/v5: Get "https://proxy.golang.org/github.com/jackc/pgx/v5/@v/list": read tcp 10.11.11.69:53838->173.194.222.141:443: read: connection reset by peer

**Comment by jackc** [comment]
Seems like a network issue somewhere between you and the Go module proxy.

If the problem persists maybe try running with env `GOPROXY=off`.

**Reply by cskml** [reply]
Thank you for Ans @jackc,  I tried  GOPROXY with direct and off, not works. About network, checked with traceroute hops, on 3 layer ok, checked with curl "https://proxy.golang.org/github.com/jackc/pgx/v5/@v/list" -> 200 OK, ... I don't know what's wrong.

**Reply by jackc** [reply]
That's really strange. Here's a few more things you could try to narrow down the problem:

1. Try on a different computer
2. Try from a different network
3. See if you can get other Go packages from the computer and network that is currently failing to get pgx.
